src/messages/MMgrReport.h

```diff
@@ -44,14 +45,15 @@ class PerfCounterType
     // TODO: decide whether to drop the per-type
     // encoding here, we could rely on the MgrReport
     // verisoning instead.
-    ENCODE_START(3, 1, bl);
+    ENCODE_START(4, 1, bl);
     encode(path, bl);
     encode(description, bl);
     encode(nick, bl);
     static_assert(sizeof(type) == 1, "perfcounter_type_d must be one byte");
     encode((uint8_t)type, bl);
     encode(priority, bl);
     encode((uint8_t)unit, bl);
+    encode(labels, bl);
```

Comment:
Given that you have decided to punt on labels in the manager, there is no reason to attempt to encode them.